modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/liferay/surface/Surface.es.js

```diff
@@ -14,7 +14,8 @@
 
 import core from 'metal';
 import dom from 'metal-dom';
-import Surface from 'senna/lib/surface/Surface';
+
+import Surface from '../../senna/surface/Surface';
```

Comment:
Same with this... overall seems like we should flatten quite a bit of the hierarchy here (doesn't need to be in this PR, but as a follow-up). There are a bunch of folders called `foo` that each contain exactly one file called `foo` or similar.

Reply:
Yeah, I noticed... we can do it in a followup, indeed 👍